src/index.ts

```diff
@@ -102,18 +102,19 @@ const FormsortWebEmbed = (
   const onEventMessage = (eventData: IIFrameAnalyticsEventData) => {
     const { eventType } = eventData;
     if (eventType === AnalyticsEventType.FlowLoaded) {
```

Comment:
@fzembow would `switch case` look better as the conditions check a single variable here?

Reply:
Sure

Maybe also just unpack the `onX` like:

```
const {
  onFlowLoaded,
  onFlowFinalized,
  // ...
} = eventListeners;
```

So that you don't keep repeating `eventListeners.*` everywhere